src/driver.coffee

```diff
@@ -51,11 +48,11 @@ class Neo4jDB
     else
       "#{value}"
 
-  reset: ->
-    console.log "Resetting Neo4j"
```

Comment:
when doing pull requests, it best to be simple and clean. Look at all these changes that arent changes. I'm not trying to be too hard on you, but its good practice to get in to -- especially if youre collaborating. Check out GitUp. Its open source and works great for breakin down your changed into multiple commits.